SECURITY.md

```diff
@@ -2,12 +2,62 @@
 
 ## Reporting a Vulnerability
 
-For general cFS vulnerabilities please [open a cFS framework issue](https://github.com/nasa/cfs/issues/new/choose).
+For general cFS vulnerabilities, please [open a cFS framework issue](https://github.com/nasa/cfs/issues/new/choose). Please use the "Bug Report" template and provide as much information as possible. Apply appropraite labels for each report.  
 
-Please use the "Bug Report" template and provide as much information as possible. Apply appropraite labels for each report. For security related reports, tag the issue with the "security" label.
+For security related vulnerabilities, follow the general cFS vulnerabilites instructions and tag the issue with the "security" label.
+
+## Testing
+
+**Disclaimer: nasa/cFS is not responsible for any liability incurred under the [Apache License 2.0](https://github.com/nasa/cFS/blob/main/LICENSE).**
+
+Testing is an important aspect our team values to improve the cFS bundle. Several tools are used for testing:
+
+### CodeQL
+
+The [cFS CodeQL GitHub Actions workflow](https://github.com/nasa/cFS/actions/workflows/codeql-build.yml) is available to the public, however, the results are not available to the public. 
+
+CodeQL is ran for every push and pull-request on all branches of cFS in GitHub Actions. 
+
+For the CodeQL GitHub Actions setup, visit https://github.com/github/codeql-action. 
+
+### Cppcheck
+
+The [cFS Cppcheck GitHub Actions workflow and results](https://github.com/nasa/cFS/actions/workflows/static-analysis.yml) are available to the public. To view the results, select a workflow and download the artifacts. 
+
+Cppcheck is ran for every push on the main branch and every pull request on all branches of cFS in Github Actions. 
+
+For more information about Cppcheck, visit http://cppcheck.sourceforge.net/.
+
+### CodeSonar
+
+CodeSonar results are not available to the public. 
+
+CodeSonar is typically ran on a “once per release” sort of schedule.  
+
+For more information about CodeSonar, visit https://www.grammatech.com/codesonar-cc. 
+
+### LGTM
```

Comment:
Do you want to move LGTM up by CodeQL since they use the same engine?

Reply:
Moved LGTM under CodeQL.